lib/src/ui/trina_body_rows.dart

```diff
@@ -224,7 +229,14 @@ class TrinaBodyRowsState extends TrinaStateWithChange<TrinaBodyRows> {
                             ),
                           // Scrollable rows
                           Expanded(
+<<<<<<< Updated upstream
                             child: ListView.builder(
+=======
+                            child: (scrollConfig.smoothScrolling
+                                ? TrinaSmoothListView.builder
+                                : ListView.builder)(
+                              cacheExtent: stateManager.rowsCacheExtent,
+>>>>>>> Stashed changes
```

Comment:
It seems this is an unresolved conflict.

Reply:
Thank you. I merged the changes, hopefully PR #76 is accepted too.